packages/legend-studio/src/index.ts

```diff
@@ -135,6 +135,11 @@ export {
   V1_propertyPointerModelSchema,
   V1_stereotypePtrSchema,
 } from './models/protocols/pure/v1/transformation/pureProtocol/serializationHelpers/V1_DomainSerializationHelper';
+export {
```

Comment:
Could you move this to the `V1` block in `DSLMapping_Exports.ts` instead of this?